aiocogeo/partial_reads.py

```diff
@@ -270,9 +270,13 @@ def _resample(
         self, clipped: NpArrayType, img_tiles: TileMetadata, out_shape: Tuple[int, int], resample_method: int
     ) -> NpArrayType:
         """Resample a numpy array to the desired shape"""
-        img = Image.fromarray(np.rollaxis(clipped, 0, 3))
-        resized = img.resize((out_shape[0], out_shape[1]), resample=Image.BILINEAR)
-        resized = np.rollaxis(np.array(resized), 2, 0)
+        _clipped = np.rollaxis(clipped, 0, 3)
+        if clipped.shape[0] == 1:
+            _clipped = np.squeeze(_clipped, axis=2)
+        img = Image.fromarray(_clipped)
+        resized = np.array(img.resize((out_shape[0], out_shape[1]), resample=Image.BILINEAR)).astype(img_tiles.dtype)
```

Comment:
should `resample=Image.BILINEAR` be `resample=resample_method` here ?

Reply:
good catch!